app/services/analytics_events.rb

```diff
@@ -671,6 +671,14 @@ def idv_doc_auth_agreement_visited(**extra)
     track_event('IdV: doc auth agreement visited', **extra) # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
   end
 
+  def idv_doc_auth_how_to_verify_visited(**extra)
+    track_event('IdV: doc auth how to verify visited', **extra) # rubocop:disable IdentityIdp/AnalyticsEventNameLinter
```

Comment:
Please use the [new format](https://github.com/18F/identity-idp/pull/9318) for new analytics events and remove the `rubocop:disable` comment.

You're also going to get a nudge from the linter to put these in alphabetical order based on method name.